lively.project/prompts.cp.js

```diff
@@ -453,7 +462,17 @@ export const SaveProjectDialog = component(SaveWorldDialog, {
           borderWidth: 0,
           position: pt(179, 0)
         }]
-      }]
+      }, part(SystemButton, {
+        name: 'diff button',
+        extent: pt(449.5, 27),
+        submorphs: [{
+          name: 'label',
+          textAndAttributes: ['Show Summary of Changes (Advanced Operation)', null, '', {
+            fontFamily: 'tabler-icons',
+            fontWeight: '900'
+          }, ' ', {}]
+        }]
+      })]
```

Comment:
Can we replace with with a dark system button? It will look better when placed on the dark prompt.

Reply:
![image](https://github.com/LivelyKernel/lively.next/assets/14252419/9ebc49e2-cbc1-4fa5-a239-4aaa624b8a32)